### DISCUSSION TITLE
Conroller trials on real crazyflie

### DISCUSSION THREAD

**Discussion by ntukenmez3** [bitcraze/discussions#945] [Q&A] [answered]
Hi all, we'd like to use our controller on a real crazyflie2 via ros2. Do you have any useful guide to make it possible, thanks.

**Comment by knmcguire** [comment, marked as answer]
Hi!

So we don't have a ROS2 guide on this ourselves. The best way is to use the [crazyswarm2 project](https://imrclab.github.io/crazyswarm2/) and use the right topics to send the resulting attitude and altitude commands. Best to use their discussion forum for that but I believe you have already found that https://github.com/IMRCLab/crazyswarm2/discussions.

**Reply by ntukenmez3** [reply]
I’m currently working on that repository. Thanks!
